src/test_driver/esp32/main/main_app.cpp

```diff
@@ -81,52 +74,6 @@ extern "C" void app_main()
         exit(err);
     }
 
-    // Initialize the LwIP core lock.  This must be done before the ESP
-    // tcpip_adapter layer is initialized.
-    err = PlatformMgrImpl().InitLwIPCoreLock();
-    if (err != CHIP_NO_ERROR)
-    {
-        ESP_LOGE(TAG, "PlatformMgr().InitLocks() failed: %s", ErrorStr(err));
-        exit(err);
-    }
-
-    err = esp_netif_init();
-    if (err != CHIP_NO_ERROR)
-    {
-        ESP_LOGE(TAG, "esp_netif_init() failed: %s", ErrorStr(err));
-        exit(err);
-    }
-
-    // Arrange for the ESP event loop to deliver events into the CHIP Device layer.
-    err = esp_event_loop_create_default();
-    if (err != CHIP_NO_ERROR)
-    {
-        ESP_LOGE(TAG, "esp_event_loop_create_default() failed: %s", ErrorStr(err));
-        exit(err);
-    }
-    esp_netif_create_default_wifi_ap();
-    esp_netif_create_default_wifi_sta();
-
-    err = esp_event_handler_register(WIFI_EVENT, ESP_EVENT_ANY_ID, PlatformManagerImpl::HandleESPSystemEvent, NULL);
-    if (err != CHIP_NO_ERROR)
-    {
-        ESP_LOGE(TAG, "esp_event_handler_register() failed for WIFI_EVENT: %s", ErrorStr(err));
-        exit(err);
-    }
-    err = esp_event_handler_register(IP_EVENT, ESP_EVENT_ANY_ID, PlatformManagerImpl::HandleESPSystemEvent, NULL);
-    if (err != CHIP_NO_ERROR)
-    {
-        ESP_LOGE(TAG, "esp_event_handler_register() failed for IP_EVENT: %s", ErrorStr(err));
-        exit(err);
-    }
-
-    err = Crypto::add_entropy_source(app_entropy_source, NULL, 16);
```

Comment:
crypto tests sound as if they may want an entropy source. Event loop also seems potentially important.

I would understand 'esp tests do not use wifi' because qemu, however this should be documented and probably a separate PR from the stack module unless it interferes with that...and if it does please explain why.

Reply:
This are duplicated in `PlatformManagerImpl::_InitChipStack`

https://github.com/project-chip/connectedhomeip/blob/82a8544c3a7df2e59e16f30fb7393df22a654a87/src/platform/ESP32/PlatformManagerImpl.cpp#L56